nemo/collections/nlp/nm/trainables/common/transformer/transformer_modules.py

```diff
@@ -22,27 +22,23 @@
 http://nlp.seas.harvard.edu/2018/04/03/attention.html
 Copyright by the HuggingFace and Annotated Transformer authors.
 """
-__all__ = [
-    'FixedPositionalEncoding',
-    'TransformerEmbedding',
-    'MultiHeadAttention',
-    'LightweightConv1d',
-    'TwoStreamSelfAttention',
-    'PositionWiseFF',
-]
+__all__ = []
```

Comment:
Why is there an empty \_\_all__ >

Reply:
I guess it is to prevent the user to import everything in that file by using import *. @ekmb @yzhang123 are these necessary?

Reply:
@blisc I think we can leave it empty because of what @VahidooX said and make the package autocompletion only list user api. Please note that users can still circumvent this by importing that entire path